ci/run_test/run_test_post_always.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# This is a post test processing script for post processing the
+# run_test.sh stage CI run
+
+set -uex
+
+# shellcheck disable=SC1091
+source ./.build_vars.sh
+
+rm -rf run_test.sh vm_test
+DAOS_BASE="${SL_PREFIX%/install*}"
+NODE="${NODELIST%%,*}"
+
+mydir="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+# shellcheck disable=SC2029
+ssh "$SSH_KEY_ARGS" jenkins@"$NODE" \
+  "DAOS_BASE=$DAOS_BASE      \
+   HOSTNAME=$HOSTNAME        \
+   PWD=$PWD                  \
+   $(cat "$mydir/run_test_post_always_node.sh")"
```

Comment:
ACtually this seems to be used in many places, would it be possible to scp the script, then run it remotely?

Reply:
It could be possible.  This was the code I got as a sample refactoring from Brian.